### PR TITLE
Fixed broken links in README.md

### LINKED ISSUE #424: Multiple links in `README.md` are broken.
/kind bug
/kind documentation

**What steps did you take and what happened:**
Multiple links are broken in the `README.md` file. You'll have to check one by one and attach the link of the mentioned file wherever necessary.

**What did you expect to happen:**
The links should direct to the mentioned page.

### PR DESCRIPTION
**Pull request by harshmathurx**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fixes broken links in the README.md file

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #424 

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [x] includes documentation
- [ ] adds unit tests

**Release note**: NONE
-->
```release-note
    NONE
```

### CODE REVIEW

**Comment by harshmathurx** [comment]
> 
> 
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
>     * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> 
>     * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> 
>     * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

Just signed the agreement :)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="capi" src="https://github.com/kubernetes-sigs/cluster-api/raw/master/docs/book/src/images/introduction.png" width="160x" /><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p>
+<p align="center"><img alt="capi" src="https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/b75062c95d283880964c0a13482bd82b7523814d/docs/book/src/images/introduction.png" width="160x" /><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p>
```
Comment (dims) [inline README.md:1]: `https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/docs/book/src/images/introduction.png` would. be better? (avoid SHA)

Comment (harshmathurx) [reply README.md:1]: Yes, I'll change that

Comment (harshmathurx) [reply README.md]: Yes, thank you.
I made that change

**Review by jayesh-srivastava — APPROVED** [review]
lgtm

**Comment by jayesh-srivastava** [comment]
@harshmathurx You'll have to rebase the PR.

**Pull request closed without merging**